src/remote/action.go

```diff
@@ -135,7 +135,11 @@ func (c *Client) getCommand(target *core.BuildTarget) string {
 		}
 		return cmd
 	}
-	return target.GetCommand(c.state)
+	cmd := target.GetCommand(c.state)
+	if target.IsBinary {
+		return "(" + cmd + ") && chmod +x $OUT"
```

Comment:
might need to be $OUTS (don't think we necessarily guarantee binary targets have a single output, although it is a good idea for them to)